src/routes/teams.js

```diff
@@ -871,45 +872,63 @@ async function inviteTeamMember(request, h) {
     const { auth, params, payload, server } = request;
 
     const isAdmin = server.methods.isAdmin(request);
-    let userWasCreated = false;
+
+    const user = auth.artifacts;
 
     if (!isAdmin) {
-        const memberRole = await getMemberRole(auth.artifacts.id, params.id);
+        const memberRole = await getMemberRole(user.id, params.id);
 
-        if (memberRole === ROLES[2]) {
+        if (memberRole === ROLES[2] || user.role === 'pending') {
             return Boom.unauthorized();
         }
     }
 
+    const maxTeamInvites = await getMaxTeamInvites({
+        server,
+        user,
+        team: await Team.findByPk(params.id)
+    });
+
+    if (maxTeamInvites !== false) {
+        const pendingInvites = await getPendingTeamInvites({ user });
+        if (pendingInvites > maxTeamInvites) {
```

Comment:
```suggestion
        if (pendingInvites >= maxTeamInvites) {
```
This is me being pedantic 😅. Do you want 20 or 21 pending invites?